src/Input/Input.js

```diff
@@ -261,6 +262,16 @@ module.exports = kind(
 
 	/**
 	* @private
+	* Fix to safari input type number allowing other characters
+	*/
+	testNumber: function(sender, e) {
+		if (this.type == 'number' && !/[eE.0-9+-]/.test(e.key)) {
```

Comment:
Let's use `===` here and place the regex outside of this statement and at the module level, so that it can be compiled ahead of time instead of created on each function call.